.circleci/config.yml

```diff
@@ -140,6 +140,33 @@ jobs:
             ./tmp/cc-test-reporter sum-coverage tmp/codeclimate.*.json -p 4 -o tmp/codeclimate.total.json
             ./tmp/cc-test-reporter upload-coverage -i tmp/codeclimate.total.json
 
+  update_spree_designs_emails:
+    docker:
+      - image: *image
+    working_directory: ~/spree
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "0b:96:15:2d:34:4d:bc:75:d9:fc:7f:1b:de:a5:70:d5"
+      - checkout
```

Comment:
Do we need this step? We're not doing anything with `spark-solutions/spree` repo.

Reply:
thanks, I found a more elegant way to add host to `known_hosts`
removed `checkout` step